core/shared/src/main/scala/org/typelevel/log4cats/natchez/package.scala

```diff
@@ -0,0 +1,82 @@
+package org.typelevel.log4cats
+
+import _root_.natchez._
+import cats._
+import cats.syntax.all._
+import org.typelevel.log4cats.extras.LogLevel
+
+package object natchez {
+  /**
+   * `LogLevel` as an OpenTelemetry `SeverityNumber`
+   *
+   * See https://opentelemetry.io/docs/reference/specification/logs/data-model/#field-severitynumber
+   *
+   * @return LogLevel as an OpenTelemetry SeverityNumber, converted to the TraceValue ADT
+   */
+  private val logLevelToSeverityNumber: LogLevel => TraceValue = {
```

Comment:
Yeah but that would require giving a name to something that doesn't need it 😭

Do you think it would make a measurable difference? If so, I can update all three of those function vals.

Reply:
> Do you think it would make a measurable difference?

On in microbenchmarks :trollface: